jgrapht-core/src/main/java/org/jgrapht/generate/NamedGraphGenerator.java

```diff
@@ -0,0 +1,1033 @@
+            addEdge(targetGraph, edge[0], edge[1]);
+    }
+
+    //-------------Clebsch Graph-----------//
+    /**
+     * @see #generateClebschGraph
+     * @return Clebsch Graph
+     */
+    public static Graph<Integer, DefaultEdge> clebschGraph(){
+        Graph<Integer, DefaultEdge> g=new SimpleGraph<>(DefaultEdge.class);
+        new NamedGraphGenerator<Integer, DefaultEdge>(new IntegerVertexFactory()).generateClebschGraph(g);
+        return g;
+    }
+    /**
+     * Generates a <a href="http://mathworld.wolfram.com/ClebschGraph.html">Clebsch Graph</a>.
+     * The Clebsch graph, also known as the Greenwood-Gleason graph (Read and Wilson, 1998, p. 284), is a strongly
+     * regular quintic graph on 16 vertices and 40 edges.
+     * @param targetGraph receives the generated edges and vertices; if this is non-empty on entry, the
+     *        result will be a disconnected graph since generated elements will not be connected to
+     *        existing elements
+     */
+    public void generateClebschGraph(Graph<V,E> targetGraph){
+        vertexMap.clear();
+        int x = 0;
+        for(int i=0; i<8; i++) {
+            addEdge(targetGraph, x % 16, (x + 1) % 16);
+            addEdge(targetGraph, x % 16, (x + 6) % 16);
+            addEdge(targetGraph, x % 16, (x + 8) % 16);
+            x++;
+            addEdge(targetGraph, x % 16, (x + 3) % 16);
+            addEdge(targetGraph, x % 16, (x + 2) % 16);
+            addEdge(targetGraph, x % 16, (x + 8) % 16);
+            x++;
+        }
+    }
+
+    //-------------Grötzsch Graph-----------//
+    /**
+     * @see #generateGrötzschGraph
+     * @return Grötzsch Graph
+     */
+    public static Graph<Integer, DefaultEdge> grötzschGraph(){
+        Graph<Integer, DefaultEdge> g=new SimpleGraph<>(DefaultEdge.class);
+        new NamedGraphGenerator<Integer, DefaultEdge>(new IntegerVertexFactory()).generateGrötzschGraph(g);
+        return g;
+    }
+    /**
+     * Generates a <a href="http://mathworld.wolfram.com/GroetzschGraph.html">Grötzsch Graph</a>.
+     * The Grötzsch graph is smallest triangle-free graph with chromatic number four.
+     * @param targetGraph receives the generated edges and vertices; if this is non-empty on entry, the
+     *        result will be a disconnected graph since generated elements will not be connected to
+     *        existing elements
+     */
+    public void generateGrötzschGraph(Graph<V,E> targetGraph){
+        vertexMap.clear();
+        for(int i=1; i<6; i++)
+            addEdge(targetGraph, 0, i);
+        addEdge(targetGraph, 10, 6);
+        for(int i=6; i<10; i++){
+            addEdge(targetGraph, i, i+1);
+            addEdge(targetGraph, i, i-4);
+        }
+        addEdge(targetGraph, 10, 1);
+        for(int i=7; i<11; i++)
+            addEdge(targetGraph, i, i-6);
+        addEdge(targetGraph, 6, 5);
+    }
+    ///////////////////////////////////////////////
+
+    //-------------Hall-Janko Graph-----------//
```

Comment:
@jsichi Ideally that graph would indeed be represented by a sparse6 or graph6 description which is pretty concise. We could then use the corresponding importer to read the graph. The problem is however that all importers are in the jgrapht-io module, and the graph generators (including this one) are in the jgrapht-core module. To access the importers from the jgrapht-core module, a dependency on jgrapht-io would be required. However, in the previous release, the IO module was explicitly created to remove this dependency. So I'm not sure how to deal with this. Any recommendations?